Mathlib/Analysis/SpecialFunctions/Integrability/Basic.lean

```diff
@@ -252,6 +252,6 @@ theorem intervalIntegrable_one_div_one_add_sq :
 @[simp]
 theorem intervalIntegrable_inv_one_add_sq :
     IntervalIntegrable (fun x : ℝ => (↑1 + x ^ 2)⁻¹) μ a b := by
-  field_simp; exact mod_cast intervalIntegrable_one_div_one_add_sq
+  field_simp [mod_cast intervalIntegrable_one_div_one_add_sq]
```

Comment:
With the impending rewrite of field_simp, these steps may just create superfluous churn. I'm not sure if they are worthwhile. (Also for the other instance below.)